src/buildCommon.js

```diff
@@ -116,13 +139,19 @@ var mathit = function(value, mode, options, classes) {
     if (/[0-9]/.test(value.charAt(0)) ||
             // glyphs for \imath and \jmath do not exist in Math-Italic so we
             // need to use Main-Italic instead
-            utils.contains(mainitLetters, value) ||
-            utils.contains(greekCapitals, value)) {
-        return makeSymbol(
-            value, "Main-Italic", mode, options, classes.concat(["mainit"]));
+            utils.contains(mainitLetters, value)) {
+            // For greek capitals, we have a choice, but Math-Italic
+            // seems closer to LaTeX.
+            //|| utils.contains(greekCapitals, value)) {
```

Comment:
In that case, can you remove the commented out code?  TBH, I'm not sure why we have the same glyphs duplicated in different fonts.  It would be nice if we could avoid this and save a few bytes.  I'm curious to find out why the rendering looks different even though the glyphs are the same.  Maybe the advance or skew metrics for the Greek capitals are differ between fonts.  I think it's safe to investigate this separate from this pull request.